records/filter.record.ts

```diff
@@ -0,0 +1,141 @@
+import { pool } from "../config/db";
+import { AvailableStudent, FilterEntity } from "../types";
+import { FieldPacket } from "mysql2";
+
+
+type AvailableStudentResults = [AvailableStudent[], FieldPacket[]];
+type AllRecordsStudentResults = [{totalCount:number}[],FieldPacket[]];
+
+export class FilterRecord implements FilterEntity{
+    remoteWork: boolean|string;
+    inOffice: boolean|string;
+    employmentContract: boolean|string;
+    b2b: boolean|string;
+    mandateContract: boolean|string;
+    workContract: boolean|string;
+    min: string|number;
+    max: string|number;
+    canTakeApprenticeship: boolean|null|string;
+    monthsOfCommercialExp: string|null;
+    courseCompletion: string|number;
+    courseEngagement: string|number;
+    projectDegree: string|number;
+    teamProjectDegree: string|number;
+    page: string|number;
+    rowsPerPage : string|number;
+    hrId?: string;
```

Comment:
Pisałem już na FE, że nie wiem czemu to jest traktowane jako "redkord". Filtr wcale nim nie jest, student jest. To trochę mylące.